addons/mail/static/tests/owl/component/discuss_tests.js

```diff
@@ -4453,134 +4453,117 @@ QUnit.test('moderation: moderated channel with pending moderation message', asyn
     );
 });
 
-QUnit.skip('moderation: accept pending moderation message', function (assert) {
-    // TODO FIXME converting this test is a waste of time, there are bigger priorities, task-2199306
+QUnit.test('moderation: accept pending moderation message', async function (assert) {
     assert.expect(12);
-    var done = assert.async();
 
-    this.data.initMessaging = {
+    const moderatedChannelId = 20;
```

Comment:
keep hard-coded value instead of storing in a variable
_(same comment in other unskipped tests)_

Reply:
done